lib/kubeclient.rb

```diff
@@ -13,6 +13,9 @@ module Kubeclient
   class Client < Common::Client
     attr_reader :api_endpoint
 
+    SERVICE_ACCOUNT_TOKEN_FILE =
```

Comment:
> Well OK - it's your client after all... I can just see this leading to lots of wasted time by devs using this as they hunt down typos in the token file name & actually finding the token file name... but I''ll remove it at your request.

thank you :)
This token/path should be available and documented on server side docs.
plus, as we discussed in the past, using this from inside k8s is not the main use case, hence I currently do not expect a lot of issues around this.
And as in any issue and design, this can be always reevaluated and change in the future if required.